packages/expo-location/src/ExpoLocation.web.ts

```diff
@@ -72,6 +72,9 @@ export default {
   async watchDeviceHeading(headingId): Promise<void> {
     console.warn('Location.watchDeviceHeading: is not supported on web');
   },
+  async hasServicesEnabledAsync(): Promise<boolean> {
+    return false;
```

Comment:
`return 'geolocation' in navigator;` will be better here. See https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API#The_geolocation_object
It's the same value as returned by `(await getProviderStatusAsync()).locationServicesEnabled` - I just wanted to make more convenient API for checking this thing 😉